contracts/MultiSigWallet.sol

```diff
@@ -194,6 +195,25 @@ contract MultiSigWallet {
         confirmTransaction(transactionId);
```

Comment:
I think we do not need submitTransaction any longer

Reply:
Decided to support multisig interface.